helmchart/values.yaml

```diff
@@ -37,20 +37,26 @@ images:
     imageName: redis
     tag: latest
     imagePullPolicy: Always
-  db:
-    imageName: dev_vro-postgres
-    tag: 0b9c9c4
```

Comment:
@advcoder I assumed this should have:
```
    imagePullPolicy: Always
```
since it was converted into a subproject in the `postgres` folder. Do you agree? I moved this section lower for consistency.